src/widget/tooltipqopengl.cpp

```diff
@@ -0,0 +1,46 @@
+#include "widget/tooltipqopengl.h"
+
+#include <widget/wglwidget.h>
+
+#include <QStyle>
+#include <QTimer>
+#include <QToolTip>
+
+ToolTipQOpenGL::ToolTipQOpenGL()
+        : m_timer(new QTimer()) {
+    m_timer->setSingleShot(true);
+    connect(m_timer.get(), SIGNAL(timeout()), this, SLOT(onTimeout()));
```

Comment:
```suggestion
    connect(m_timer.get(), SIGNAL(timeout()), this, SLOT(onTimeout()));
```
Please fix this Clazy warning, by using the modern connect syntax without SIGNAL and SLOT macros.

Reply:
Thanks for reviewing. I will remove the SIGNAL / SLOT macros. I will submit a fix for the debug assert in a moment, it's something silly (I forgot to run with a debug build).

Reply:
done